g_chi_docs.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "strings"
+
+func isCHI(controllerName string) bool {
+	if controllerName == "" {
+		return true
+	}
+
+	if !strings.HasSuffix(controllerName, "Controller") {
+		return true
+	}
```

Comment:
Is this something that we've agreed upon?? :thinking: 

I believe we should inform other about Controller as prefix

Reply:
Oh wait, I misunderstood this.... but yeah, maybe someday someone uses Controller as suffix. On this case, it would be skipped, right?

Reply:
Yeah, you're right. But that would be against the microservice template and our voting results, no? 🤔

Reply:
I don't think we have voted for the naming :sweat_smile: 
But yeah, is there any other way to identify the go-chi controller without checking for the name??

Reply:
ah ok sorry, seems I confused myself, I was talking about this decision: https://zalora.atlassian.net/wiki/spaces/MB/pages/3566174210/Location+of+the+new+HTTP+handlers, but it does not mention anything about struct related name.

> is there any other way to identify the go-chi controller without checking for the name??

now that I just read the decision doc, we could maybe use `pkgpath` here instead of controllerName. So we could just check for the `github.com/zalora/doraemon/handlers/...` prefix to determine if it's chi or beego. WDYT?

Reply:
Noice, I believe we can use it.

Reply:
Cool! Addressed in https://github.com/zalora/bee/pull/30/commits/e5828447a2391bf0ae05ffaacdeb2da1e0b34d73